packages/pulp-deb/pulp-deb.spec

```diff
@@ -9,12 +9,12 @@
 
 # ---- archive related macros ----
 
-%define git_tag %{version}rc1
+%define git_tag %{version}rc2
 %define srcname pulp_deb
 
 Name: pulp-deb
 Version: 1.9.0
-Release: 0.1.rc%{?dist}
+Release: 0.2.rc%{?dist}
```

Comment:
Why the release bump?

Reply:
@evgeni Because I messed up the 1.9-release branch, and branched it from an old HEAD.  This wasn't discovered untill after I built the release bits in koji.  I had to rebase, and since 0.1.rc is taken, need a new 0.2.rc

Reply:
Gotcha